lib/winston/logger.js

```diff
@@ -107,6 +107,12 @@ class Logger extends Transform {
     // Hoist other options onto this instance.
     this.levels = levels || this.levels || config.npm.levels;
     this.level = level;
+    if(typeof this.exceptions !== 'undefined') {
+      this.exceptions.unhandle();
+    }
+    if(typeof this.rejections !== 'undefined') {
```

Comment:
This check isn't great. If it were more generic, it would be more foolproof. This handles the case where this is `false`, `null`, or `undefined`.

```suggestion
    if (this.exceptions) {
      this.exceptions.unhandle();
    }
    if (this.rejections) {
```